src/main/kotlin/org/wfanet/measurement/kingdom/db/testing/AbstractKingdomRelationalDatabaseTest.kt

```diff
@@ -61,12 +61,13 @@ val REFUSAL: Refusal = Refusal.newBuilder().apply {
 abstract class AbstractKingdomRelationalDatabaseTest {
   /** [KingdomRelationalDatabase] instance. */
   abstract val database: KingdomRelationalDatabase
+  abstract val databaseTestHelper: DatabaseTestHelper
```

Comment:
Not sure if we want to put those last 2 methods (createNextReport, createRequisition) inside DatabaseTestHelper as well? Then we can get rid of the KingdomRelationalDatabase here.

Reply:
It's an interesting idea, but I think I want to keep each method defined in one place. Since `createRequisition` and `createNextReport` are still used by non-test code, let's keep them in one place. When we migrate to v2alpha, both of these methods will need to change anyway.

Reply:
Sounds good.